boards/arm64/qemu_cortex_a53/board.cmake

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) 2019 Carlo Caione <ccaione@baylibre.com>
+#
+# SPDX-License-Identifier: Apache-2.0
+
+set(EMU_PLATFORM qemu)
+set(QEMU_ARCH aarch64)
+
+set(QEMU_CPU_TYPE_${ARCH} cortex-a53)
+set(QEMU_FLAGS_${ARCH}
+  -cpu ${QEMU_CPU_TYPE_${ARCH}}
+  -nographic
+  -machine virt
```

Comment:
I think it would benefit from targeting a real hardware platform such as xilinx_zynqmp instead, since it can be used as a reference platform for functional verification if ever necessary.

In fact, all the other qemu_cortex_* emulated boards are targeting a qemu machine that has a real hardware equivalent.